src-web/hooks/useSidebarItemCollapsed.ts

```diff
@@ -41,3 +41,16 @@ export function getSidebarCollapsedMap() {
 
   return value;
 }
+
+export async function expandFolder(folderId: string) {
+  if (!folderId) return;
+
+  const workspaceId = jotaiStore.get(activeWorkspaceIdAtom);
+  const collapsedMap = getSidebarCollapsedMap();
+
```

Comment:
We should probably fetch the folder from there store here and recursively expand all folders up the chain too, in case of multiple nested folders.